catalog/app/containers/App/App.js

```diff
@@ -35,29 +35,36 @@ const LegacyPackages = ({ location: l }) => {
   return <AbsRedirect url={urls.legacyPackages(legacyPackagesRedirect, l)} />
 }
 
-const mkLazy = (load) => loadable(load, { fallback: () => <Placeholder /> })
-
-const Admin = mkLazy(() => import('containers/Admin'))
-const AuthActivationError = mkLazy(() => import('containers/Auth/ActivationError'))
-const AuthCode = requireAuth()(mkLazy(() => import('containers/Auth/Code')))
-const AuthPassChange = mkLazy(() => import('containers/Auth/PassChange'))
-const AuthPassReset = mkLazy(() => import('containers/Auth/PassReset'))
-const AuthSignIn = mkLazy(() => import('containers/Auth/SignIn'))
-const AuthSignOut = mkLazy(() => import('containers/Auth/SignOut'))
-const AuthSignUp = mkLazy(() => import('containers/Auth/SignUp'))
-const AuthSSOSignUp = mkLazy(() => import('containers/Auth/SSOSignUp'))
-const Bucket = mkLazy(() => import('containers/Bucket'))
-const Search = mkLazy(() => import('containers/Search'))
-const UriResolver = mkLazy(() => import('containers/UriResolver'))
-
-const Landing = mkLazy(() => import('website/pages/Landing'))
-const OpenLanding = mkLazy(() => import('website/pages/OpenLanding'))
-const OpenProfile = requireAuth()(mkLazy(() => import('website/pages/OpenProfile')))
-const Install = mkLazy(() => import('website/pages/Install'))
-
-const MAbout = mkLazy(() => import('website/pages/About'))
-const MPersonas = mkLazy(() => import('website/pages/Personas'))
-const MProduct = mkLazy(() => import('website/pages/Product'))
+const Admin = RT.mkLazy(() => import('containers/Admin'), Placeholder)
+const AuthActivationError = RT.mkLazy(
+  () => import('containers/Auth/ActivationError'),
+  Placeholder,
+)
+const AuthCode = requireAuth()(
+  RT.mkLazy(() => import('containers/Auth/Code')),
+  Placeholder,
```

Comment:
is there a typo here? like, `Placeholder`, but in other places you call `mkLazy` with two args: `() => import()` and `Placeholder`. is this intentional or a typo?

Reply:
Typo, thanks